keyboards/ergodox_ez/keymaps/rmw/tapdances.cpp

```diff
@@ -0,0 +1,465 @@
+// tapdances.cpp
```

Comment:
Could you rename this to just `tapdances.c`? I don't see any C++ code in here.

Also, I think there may be an opportunity for deduplication between this and the Kyria one, with the Userspace feature: https://docs.qmk.fm/#/feature_userspace

Reply:
Is that okay to do? I consider myself to be still pretty novice at both C and C++ (despite using them for work...). 

I agree about userspace, and I just had started trying it out, but hadn't been able to get it working yet. I'll have another crack at it.

Reply:
Done! And I moved it to the userspace